index.html

```diff
@@ -1,12 +1,1449 @@
+                </defs>
+            </svg>
+        </div>
+        <ul class="menu-items">
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle">Audit</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Manage</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Secure</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Automate</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">Resource</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="/">About</a>
+            </li>
+            <li> <button class="btn">Book a Demo</button> </li>
+        </ul>
+    </header>
+    <main>
+        <div class="services d-flex align-items-center">
```

Comment:
Move styling into relevant classes in code.

Reply:
in tabs and there the relevangt classes were not used before i have done it